pandas-stubs/core/computation/ops.pyi

```diff
@@ -34,7 +34,6 @@ class Term:
     def ndim(self) -> int: ...
 
 class Constant(Term):
-    def __init__(self, value, env, side=..., encoding=...) -> None: ...
```

Comment:
Enabled the check to sync `__init__` and `__new__` in the hope of understanding some of pyright's behavior.